dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task04320WorkflowActionRemoveNextStepConstraint.java

```diff
@@ -0,0 +1,74 @@
+package com.dotmarketing.startup.runonce;
+
+import com.dotcms.repackage.org.apache.commons.lang.StringUtils;
+import com.dotcms.util.ConversionUtils;
+import com.dotmarketing.common.db.DotConnect;
+import com.dotmarketing.common.db.DotDatabaseMetaData;
+import com.dotmarketing.common.db.ForeignKey;
+import com.dotmarketing.db.DbConnectionFactory;
+import com.dotmarketing.db.HibernateUtil;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.dotmarketing.db.HibernateUtil'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)